### PR TITLE
PP-10983- Remove JUnit 4 unit tests and replace it with JUnit 5

### PR DESCRIPTION
**Pull request by AbdirizakIdris**
- Updated JUnit4 tests to JUnit 5 equivalent

### CODE REVIEW

File: src/test/java/uk/gov/pay/connector/tasks/ParityCheckServiceTest.java [hunk 1]
```diff
@@ -55,9 +52,9 @@
 import static uk.gov.pay.connector.refund.model.domain.RefundStatus.REFUNDED;
 import static uk.gov.pay.connector.refund.model.domain.RefundStatus.REFUND_ERROR;
 import static uk.gov.pay.connector.wallets.WalletType.APPLE_PAY;
+import static uk.gov.service.payments.commons.model.Source.CARD_PAYMENT_LINK;
 
-@RunWith(MockitoJUnitRunner.class)
-public class ParityCheckServiceTest {
+@ExtendWith(MockitoExtension.class)public class ParityCheckServiceTest {
```
Comment (alexbishop1) [inline src/test/java/uk/gov/pay/connector/tasks/ParityCheckServiceTest.java]: Want a line break in here.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks good!

There’s a few places where it looks like `public class` has been replaced by `class` and `public void` has been replaced by `void` but spaces have been left before `class` and `void`, which messes up the indentation (most notable where there’s `void` immediately under `@Test`). Could you fix that please?

File: src/test/java/uk/gov/pay/connector/tasks/ParityCheckServiceTest.java (hunk 1 above)
Comment (AbdirizakIdris) [reply src/test/java/uk/gov/pay/connector/tasks/ParityCheckServiceTest.java]: Yes good spot Alex.

**Review by alexbishop1 — APPROVED** [review]
The alignment now aligns with expectations!

**Pull request merged**